dev/release/02-source.sh

```diff
@@ -17,6 +17,10 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+: ${PREPARE_DEFAULT:=1}
+: ${PREPARE_RELEASE_TAG:=${PREPARE_DEFAULT}}
+: ${PREPARE_C_GLIB_TAG:=${PREPARE_DEFAULT}}
```

Comment:
Is `_TAG` needed? It seems that we don't prepare tag for C GLib.

Reply:
The name of this environment variables was not appropriate. I've removed `_TAG`.